docs/build/messages/reaction.mdx

```diff
@@ -98,7 +98,12 @@ var registry = CodecRegistry()..registerCodec(ReactionCodec());
 
 <TabItem value="rn" label="React Native"  attributes={{className: "rn_tab"}}>
 
-React Native SDK already has the codecs registered.
+```jsx
+const client = await Client.create(wallet, {
+  env: "production",
+  codecs: [new XMTP.ReactionCodec()],
```

Comment:
Depending on the import you should hopefully not need the XMTP part lets just remove it for demo purposes.

```suggestion
  codecs: [new ReactionCodec()],
```